go/engine/identify2_test.go

```diff
@@ -665,6 +665,33 @@ func TestIdentify2NoSigchain(t *testing.T) {
 	}
 }
 
+// TestIdentifyDeleted tests that identifying a deleted user gives the correct
+// error
+func TestIdentify2Deleted(t *testing.T) {
+	tc := SetupEngineTest(t, "Identify2Deleted")
+	defer tc.Cleanup()
+
+	u := CreateAndSignupFakeUserWithUser(tc, "login")
+	userId := u.User.GetUID()
+	DeleteAccount(tc, u)
+
```

Comment:
account name shouldn't be significant.

I think we're going to need to do this for real.  Can you create a core ticket?  This should all be in one LoginState request with a session reader on the apiarg, etc.